service_mesh/service_mesh_install/installing-ossm.adoc

```diff
@@ -46,10 +42,10 @@ include::modules/ossm-cr-threescale.adoc[leveloffset=+2]
 
 include::modules/ossm-mixer-policy.adoc[leveloffset=+1]
 
-include::modules/ossm-control-plane-deploy.adoc[leveloffset=+1]
+include::modules/ossm-member-roll.adoc[leveloffset=+1]
 
-include::modules/ossm-control-plane-verify.adoc[leveloffset=+2]
+include::modules/ossm-control-plane-deploy.adoc[leveloffset=+1]
```

Comment:
This section should come before the ossm-cr sections above.

Reply:
This is the assembly file that pulls together the modules. Please see the preview attached to this PR to see how the modules are arranged.